specification/security/resource-manager/Microsoft.Security/preview/2022-04-01-preview/query.json

```diff
@@ -0,0 +1,231 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2022-04-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Security/query": {
+      "post": {
+        "tags": [
+          "Query"
+        ],
+        "description": "Run KQL query against Microsoft.Security data types for scopes specified in the request. Supported types: Nodes, edges, insights, recommendations, vulnerabilities.",
+        "operationId": "Query",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "name": "query",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/QueryRequest"
+            },
+            "description": "Request specifying query, scope and its options."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Result of the query operation",
+            "schema": {
+              "$ref": "#/definitions/QueryResponse"
+            }
+          },
+          "default": {
+            "description": "An error occurred while processing the request. See the error.code parameter to identify the specific error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Basic query": {
+            "$ref": "./examples/BasicQuery.json"
+          },
+          "Complex query": {
+            "$ref": "./examples/ComplexQuery.json"
+          }
+        }
+      }
+    },
+  },
+  "definitions": {
+    "QueryRequest": {
+      "description": "Describes a query to be executed.",
+      "type": "object",
+      "properties": {
+        "scopes": {
+          "description": "The scopes in which the query will be executed - List of subscription IDs for Azure environments, and list of Account IDs for AWS environments.",
```

Comment:
Since the description indicates that this is an 'interesting' object containing a mix of both, I think that an example that shows this type of usage would be helpful